.github/workflows/rust.yml

```diff
@@ -178,34 +179,11 @@ jobs:
       - name: Checkout
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
 
-      # On macOS, we need a proper Clang version, not Apple's custom version without wasm32 support
-      - name: Install LLVM and Clang for macOS
```

Comment:
why is this removed? Do we not run the macos clippy anymore ?
shouldn't we though ?

Reply:
The job containing this step is `cargo-runtime-build:`, it is hidden by GitHub because those lines aren't changed. We have never built runtimes on every OS, because they are all cross compiled into WASM anyway.

Clippy still runs on every OS.